Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs

```diff
@@ -22,7 +22,127 @@ private static void Test(string args, int expected)
 
         public static int FindDigit(string equation)
         {
-            // Add your code here.
+            String[] array = Regex.Split(equation, "=|\\*");   //here we split the equation into three parts
+
+            int[] numInt = new int[3];         //here we made an array for basically storing at which part is an '?'
+            for (int i = 0; i < numInt.Length; i++)
+            {
+                numInt[i] = -1;                  //initially we initialzie the array by -1 
+            }
+
+            int k = 0;
+            for (int i = 0; i < array.Length; i++)
+            {
+                String word = array[i];
+                int flag = 0;
+
+                for (int j = 0; j < word.Length; j++)
+                {
+                    if (word[j] == '?')
+                    {
+                        flag = 1;
+                        break;
+                    }
+                }
+                if (flag == 0)
+                {
+                    numInt[k++] = Int32.Parse(word);    //if the string doesn't contains '?' than we convert the string into integer
+                }                                       // and stores it in array
+                else
+                {
+                    numInt[k++] = -1;                   //if the string contains '?' than it remains -1 so that
+                }                                       //we can differentiate between three index
+            }
+            int num1 = numInt[0];
+            int num2 = numInt[1];
+            int num3 = numInt[2];
+            int mul, div, ans = 0;
+            if (num3 == -1)

```

Comment:
Try to reduce the number of if/else, as it makes it difficult to read your code.